mainnet_contract/scripts/deployment.js

```diff
@@ -28,8 +28,15 @@ async function main() {
   const data0 = await block0res.json();
   const data1 = await block1res.json();
 
-  const data0Encoded = "0x" + data0["result"]["HexRLP"]
-  const data1Encoded = "0x" + data1["result"]["HexRLP"]
+  if (!data0["result"]["Committed"] || !data0["result"]["Committed"]) {
```

Comment:
the second condition should be data1
Also, do you wanna check if data0["result"] exist first? otherwise exception will be thrown

Reply:
@wjrjerome  if data0["result"] does not exist, because the exception will be easy to find, so i have not to add this.

 i think this is a good idea, I will add it later

Reply:
Committed is a important bug , so i add in this branch